### PR TITLE
Fix/dataelements with no override

### LINKED ISSUE #75: Set override category combination to "no override"

### PR DESCRIPTION
**Pull request by tokland**
Closes #75

### CODE REVIEW

File: src/models/DataSetStore.js [hunk 1]
```diff
@@ -9,6 +22,7 @@ export default class DataSetStore {
         this.config = {
             categoryProjectsId: "MRwzyV0kXv9",
             categoryComboId: "GmXXE8fiCK5",
+            categoryComboDefaultId: "eCUWK7qTyeW",
```
Comment (adrianq) [inline src/models/DataSetStore.js]: @tokland Why do we need this?

File: src/models/DataSetStore.js [hunk 2]
```diff
@@ -126,12 +141,61 @@ export default class DataSetStore {
         this.updateFromAssociations(fieldPath, oldValue);
     }
 
+    saveDataset() {
+        return new Promise(async (complete, error) => {
+            // maintenance-app uses a custom function to save the dataset as it needs to do
+            // some processing  not allowed by dataset.save(). The code in this method is copied
+            // from maintenance-app/src/EditModel/objectActions.js
```
Comment (adrianq) [inline src/models/DataSetStore.js]: @tokland and what is the difference from what we have before? There were some properties that were not being saved before?

File: src/models/DataSetStore.js (hunk 1 above)
Comment (tokland) [inline src/models/DataSetStore.js]: Indeed, it was used at some point in my tests,  but not anymore, removed: 793da63a

**Comment by tokland** [comment]
8b48db43 sets renderAsTabs=TRUE

**Comment by adrianq** [comment]
Sweet, ready to merge then?

File: src/models/DataSetStore.js (hunk 2 above)
Comment (tokland) [inline src/models/DataSetStore.js]: The most relevant part is how dataSetElements is saved, it need the dataset ID to be specified: `dataSet: { ...dataSet, id: dataSet.id || dataSetPayload.id },`. So it gets the UID from the server before saving, assigns the datSetElements (thus they are correctly set as "no override", and then the record is saved.

**Comment by tokland** [comment]
Yup, ready to merge.

File: src/models/DataSetStore.js (hunk 2 above)
Comment (adrianq) [inline src/models/DataSetStore.js]: ahhhh... well, what a workaround anyway! Good that you catch it :)

**Comment by adrianq** [comment]
Thanks @tokland!

**Pull request merged**